google/cloud/dataproc_spark_connect/session.py

```diff
@@ -280,23 +280,23 @@ def create_session_pbar():
                                 f"Exception while writing active session to file {file_path}, {e}"
                             )
                 except (InvalidArgument, PermissionDenied) as e:
-                    stop_create_session_pbar = True
+                    stop_create_session_pbar_event.set()
```

Comment:
This is redundant given the same call in the `finally` block.

Reply:
Yes, but in catch we wait for pbar thread to stop, so we error output does not interfere with pbar - it will be too late to join the pbar thread in the finally block.